src/bidiMapper/modules/network/NetworkProcessor.ts

```diff
@@ -461,6 +458,14 @@ export class NetworkProcessor {
       }
     });
   }
+
+  static handleInterceptionError(error: any) {
```

Comment:
The naming is a bit confusing, as `handle` implies it is not re-thrown. Perhaps `wrapInterceptionErrorToBiDi`? Or return an error from this method, and re-throw it when used.

Reply:
Update. I think `ToBiDi` is redundant and not fully true as in case of other error we don't wrap it.